simapp/app_test.go

```diff
@@ -45,3 +48,82 @@ func TestGetMaccPerms(t *testing.T) {
 	dup := GetMaccPerms()
 	require.Equal(t, maccPerms, dup, "duplicated module account permissions differed from actual module account permissions")
 }
+
+func TestRunMigrations(t *testing.T) {
```

Comment:
this is a functional tests. Shall we add:
```
    if testing.Short() {
        t.Skip("skipping test in short mode.")
    }
```

Reply:
not a bad idea to distinguish short and verbose tests going forward...

Reply:
I agree. Though the `Makefile` currently does not support `-short`

Reply:
We should park this aside for a second, identify all the various test types (short, verbose, etc), introduce the necessary changes in the `Makefile` (I'm happy to take care of it), and take it from there

Reply:
@alessio thanks, sounds good. In this case, I didn't add `testing.Short()` in this test.

Reply:
If this tests runs below 0.1s then fine. Otherwise let's add `TODO: check testing.Short() flag`. 100 tests of 0.1 already give 10s ;)